clients/src/main/java/org/apache/kafka/clients/ClientUtils.java

```diff
@@ -40,8 +40,12 @@
             if (url != null && url.length() > 0) {
                 String host = getHost(url);
                 Integer port = getPort(url);
-                if (host == null || port == null)
+                if (host == null)
                     throw new ConfigException("Invalid url in " + CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG + ": " + url);
```

Comment:
I can change it if you prefer that one. I assume though that it's pretty rare for users to forget about the host part of the url, the issue there is probably some invalid character or formatting.